docs/release.rst

```diff
@@ -32,7 +32,7 @@ How to make a psycopg release
   .. __: https://github.com/psycopg/psycopg/actions/workflows/packages-bin.yml
   .. __: https://github.com/psycopg/psycopg/actions/workflows/packages-pool.yml
 
-- Delete the ``wheelhouse`` directory there is one.
+- Delete the ``wheelhouse`` directory if there is one.
```

Comment:
or this "if" is put wrong? Sorry for my English.

Reply:
It's right, thank you, it was a typo :slightly_smiling_face: